standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreDirectSql.java

```diff
@@ -1336,7 +1337,8 @@ public void visit(LeafNode node) throws MetaException {
         return;
       }
 
-      String colTypeStr = ColumnType.getTypeName(partitionKeys.get(partColIndex).getType());
+      FieldSchema partCol = partitionKeys.get(partColIndex);
```

Comment:
I don't think this variable is not being used anywhere else except L#1341, so I think it is ok to keep L1339 without any changes.